src/utils/roofline_calc.py

```diff
@@ -338,37 +336,38 @@ def calc_ai(sort_type, ret_df):
             )
             pass
         try:
-            if df.keys().str.contains("TCC_BUBBLE").sum() > 0:
-                # MI300 uses TCC_BUBBLE_sum to calculate hbm_data
+            if mspec.gpu_model == "MI200":
```

Comment:
Diff is visually messy- change made is just checking if mi200 do original calculation, otherwise do the tcc_bubble_sum calculation for mi300 and above hardware. No changes made to the actual calculations, just the if else statement.